packages/@vuepress/test-utils/lib/createJestRunner.js

```diff
@@ -5,11 +5,10 @@ const usedPorts = []
 
 module.exports = function createJestRunner (jestArgs) {
   return async function () {
-    let args = [...jestArgs, ...rawArgs]
     const execArgv = getChildProcesExecArgv()
-    args = [...execArgv, 'node_modules/.bin/jest', ...args]
+    const args = [...execArgv, ...jestArgs, ...rawArgs]
     console.log(`running node with args: ${args.join(' ')}`)
-    await execa('node', args, {
+    await execa('jest', args, {
```

Comment:
The jest runner is running under a child process, this change will cause the fail of `debug` since `execArgv` is node-level flags.

Reply:
We can merge this feature request first, and I believe this is not the real reason why Windows can't run test. Look forward to your PR later. 😉